pkg/volume/rbd/rbd.go

```diff
@@ -609,6 +610,8 @@ func (r *rbdVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 			secretName = v
 		case "usersecretnamespace":
 			secretNamespace = v
+		case "keyring":
+			keyring = v
```

Comment:
validate this is a absolute file path

Reply:
I'm having trouble finding a clean solution for this. Path formats are os dependent and I can't find a package in the standard library providing the necessary functionality. Are there any examples in the Kubernetes code?

Reply:
keyring path is on linux. A linux file path regex will do. 

This validation should go rbd volume and persistent volume source validation [functions](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go#L1185)

Admittedly, this is not the fault of this PR, it is something we should have done earlier.

Reply:
OK, thanks. I'm going to take a look at the validation package and try to add the necessary check.